server/routes/redirects.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Permanent redirects to support the website re-organization and merging of
+browser.datacommons.org with datacommons.org
+"""
+
+from flask import Blueprint
+from flask import redirect
+
+bp = Blueprint(
+  "redirects",
+  __name__,
+)
+
+
+from flask import request
+@bp.route('/kg')
+def kg():
+    return redirect('https://datacommons.org/browser', code=301)
```

Comment:
do you want to use relative path? "redirect('/browser, code=301)"

If this code is deployed to staging or prod, the /gni and /kg page will not route

Reply:
Updated to use relative paths